tests/functional/conftest.py

```diff
@@ -5,7 +5,6 @@
 import pytest
 
 from firefox_puppeteer import Puppeteer
-from firefox_puppeteer.ui.browser.window import BrowserWindow
```

Comment:
@silne30 What about where I removed it below in https://github.com/mozilla-services/services-marionette/pull/11/files/f7b5033ce06aa036a2da6748a001235707e54ca3#diff-e7c49955e98555776dc25989f7430254L6?